### PR TITLE
Added Dockerfile for whois

### PR DESCRIPTION
**Pull request by airtonzanon**

### CODE REVIEW

File: whois/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM debian:wheezy
```
Comment (jfrazelle) [inline whois/Dockerfile]: why wheezy? can it maybe be alpine :)

**Comment by airtonzanon** [comment]
I tried to use Alpine, but whois can't work.
I'll try to use scratch :D

**Comment by jfrazelle** [comment]
nah dont worry!

**Comment by jfrazelle** [comment]
oh it looks like there is a whois package in edge :)
https://pkgs.alpinelinux.org/package/edge/testing/x86_64/whois

**Comment by airtonzanon** [comment]
Hey, now it is working :D Thank you so much.
Now using the package that you told.

**Comment by jfrazelle** [comment]
merged thanks!

**Pull request merged**